packages/documentation-site/patternfly-docs/content/design-guidelines/styles/typography/typography.md

```diff
@@ -153,26 +135,22 @@ No matter which visual customizations you choose to make, you should always main
 ## Line height and spacing
```

Comment:
@lboehling is this section still meaningful given the new Figma kit or is it good to keep?